streams/jm.m3u

```diff
@@ -1,3 +1,9 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="SportsMax.jm" status="timeout",SportsMax (720p)
 http://cdn.tvmatic.net/sport.m3u8
+#EXTINF:-1 tvg-id="CaribbeanGospelTV.jm",Caribbean Gospel TV (1080p)
+https://5790d294af2dc.streamlock.net/8048/8048/playlist.m3u8
+#EXTINF:-1 tvg-id="MercyandTruthMinistriesTelevision.jm",MTM TV (720p)
+https://wowza.btbn.tv:5443/mtmtv/streams/live.m3u8
+#EXTINF:-1 tvg-id="JamaicaOnlineTV.jm",Jamaica Online TV (1080p) [Not 24/7]
+https://hls-cdn.tvstartup11.net/tvstartup11-channel/live/mp4:jotvedge/playlist.m3u8
```

Comment:
Redirects to https://vse2-eu-all14.secdn.net/tvstartup11-channel/live/mp4:jotvedge/playlist.m3u8